scripts/bme_testsuite_run.sh

```diff
@@ -0,0 +1,35 @@
+set -x 
+if [[ -z $1 || -z $2 ]]; then
+	echo "usage: bme_testsuite_run.sh [smoke|persistent-{clean,verify}] [DIR]"
+	exit 1
+fi
+
+if [[ $1 = "smoke" ]]; then
```

Comment:
Running the os-tempest-install playbook does not make `ostestr` available in the container and that still requires a pip install from the cloned tempest repo from that playbook. There needs to be a couple of additional steps to get it running, which differ slightly from the persistent tests (such as configuring the tempest.conf).